### PR TITLE
Capture authorization header for logging in endpoint requests

### LINKED ISSUE #232: Capture authentication header
Sites will include arbitrary authentication tokens that we will manually disperse to sites opting to use on, this information should be logged for metrics/logging in autoendpoint for use by metrics.

### PR DESCRIPTION
**Pull request by bbangert**
Closes #232.

@jrconlin r?

### CODE REVIEW

File: autopush/endpoint.py [hunk 1]
```diff
@@ -387,6 +387,7 @@ def _client_info(self):
             "user-agent": self.request.headers.get("user-agent", ""),
             "remote-ip": self.request.headers.get("x-forwarded-for",
                                                   self.request.remote_ip),
+            "authentication": self.request.headers.get("authentication", ""),
```
Comment (kitcambridge) [inline autopush/endpoint.py]: Isn't the header called `Authorization`?

Comment (bbangert) [inline autopush/endpoint.py]: yes, yes it is. :tired_face:

**Comment by jrconlin** [comment]
r+

**Comment by jrconlin** [comment]
Associated with #280

**Pull request merged**